finalPrep/modZeroHero.js

```diff
@@ -2,51 +2,89 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Batman"
+var specialAbility = "Extreme Wealth and Privilege"
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "Holy spamoni," +  heroName + "!"
+console.log(greeting);
+
+var catchphrase = `Alfred, the bathroom needs cleaning! I would do it but I have great responsibility due to my ${specialAbility}!`;
+console.log(catchphrase);
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 4
+var energy = 50
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower =  power * 500
+
+var fullEnergy = energy +150
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = true
+var identityConcealed = true
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ["Riddleboy", "Cat Lady", "Jared Leto"]
+var sidekicks = ["Robin", "Batgirl", "Robin 2"]
+
 // Print the first sidekick to your console
 
+console.log(sidekicks[0]);
+
 // Print the last archEnemy to the console
+console.log(archEnemies[2]);
```

Comment:
Nice work with bracket notation. A next level here could be to come up with a strategy to print the last item no matter how many elements are in the array.

Reply:
Done! I used `console.log(archEnemies.at(-1))`

Was that the sort of thing you were thinking?

Also should I be re-pushing my updated work?